Sources/TuistKit/Services/ProjectAutomation+ManifestMapper.swift

```diff
@@ -289,13 +289,19 @@ extension ProjectAutomation.BuildConfiguration.Variant {
 enum GraphServiceError: FatalError {
```

Comment:
Since you are touching this one, would you mind updating it to `LocalizedError` and turn `description` into `errorDescription: String?`? We are moving away from using `FatalError`

Reply:
You can then drop `type`.

Reply:
Sure, no problem. Like the change 👍